src/SearchPanel.jsx

```diff
@@ -8,25 +8,24 @@ import { shortIsoFormat } from './utils'
 
 export class SearchPanel extends React.Component {
   render() {
-    // Adjust UTC to local timezone
-    // This code adjusts for daylight saving time
-    // but does not work for locations east of Greenwich
-    var utcDate = new Date( this.props.lastUpdated )
-    var localTimeThen = new Date(
-      utcDate.getFullYear(),
-      utcDate.getMonth(),
-      utcDate.getDate() + 1
-    )
-
-    var tryagain = shortIsoFormat( utcDate );
-
-    return (
-      <div className="search-panel">
-        <h2>Search complaint data <span className="date-subscript">(last updated: <FormattedDate value={ tryagain } />)</span></h2>
-        <SearchBar />
-        <PillPanel />
-      </div>
-    )
+    if (this.props.lastUpdated) {
```

Comment:
You don't have to rewrite the whole HTML block, the code could be simplified to:

```Typescript
let e =  null
if ( this.props.lastUpdated ) {
  e = <span className="date-subscript">(last updated: <FormattedDate value={ this.props.lastUpdated } />)</span>
}

...

<h2>Search complaint data { e }</h2>
```